### PR TITLE
feat: dark mode support

### LINKED ISSUE #1149: Dark Mode
## User story.

As a fan of dark mode, I want dark mode everywhere, so I can always view everything in dark mode.

## Is your feature request related to a problem?

Not enough dark mode! 

## Describe the solution you'd like

Needs more dark mode. 

## Additional context

Blocked by #957.

### PR DESCRIPTION
**Pull request by mallachari**
Fixes: https://github.com/stoplightio/elements/issues/1149

Bumps JSV with fixed styles.

Peek 2021-06-02 15-45

### CODE REVIEW

**Review by philsturgeon — APPROVED** [review]

**Review by mpodlasin — COMMENTED** [review]
One little comment and waiting for @philsturgeon to comment on user usability question I had - is this story supposed to introduce some select or prop for users of elements and dev portal.

(+ StoplightProject issue)

File: packages/elements-core/.storybook/preview.jsx [hunk 1]
```diff
@@ -23,10 +22,9 @@ export const globalTypes = {
 
 const ThemeProvider = (Story, context) => {
   const theme = context.globals.theme;
+  subscribeTheme({ mode: theme === 'dark' ? 'dark' : 'light' });
```
[the remark this answers is not in the document]
Comment (mallachari) [reply packages/elements-core/.storybook/preview.jsx]: Right, it should. I've changed that. Also added it to stoplight-project. MDV looks weird as it still uses the one with ui-kit.

**Comment by philsturgeon** [comment]
We can keep this as an undocumented feature for now, as I believe @noor-ahmad and @marbemac want to rejiggle some of the dark mode colors before we get it in front of users faces. Let's progress with this when you think its ready, and we can add the component prop and document it for use later.

**Review by mpodlasin — APPROVED** [review]
In this case I am approving

**Comment by sirkadirov** [comment]
Hi! Any updates on dark mode? Thanks!

**Pull request merged**